app/Decsys/Controllers/SurveysController.cs

```diff
@@ -80,6 +80,24 @@ public IActionResult Get(
                 : Ok(survey);
         }
 
+        [HttpGet("filtering-and-sorting")]
+        [SwaggerOperation("List summary data for Surveys filtered by name and view (unarchived, archived, or all).")]
+        [SwaggerResponse(200, "A list of filtered Surveys.", typeof(IEnumerable<SurveySummary>))]
+        public IEnumerable<SurveySummary> FilteredList(
+            [FromQuery] string? name = null,
+            [FromQuery] string view = "",
+            [FromQuery] string sortBy = "name",
+            [FromQuery] string direction = "up")
```

Comment:
```suggestion
            [FromQuery] string sortBy = SurveySortingKeys.Name,
            [FromQuery] string direction = SurveySortingKeys.Direction
```
Can we update the hardcoded valued with constants declared in the `SurveySortingKeys`?